File: 2038-remove-colored-pieces-if-both-neighbors-are-the-same-color/2038-remove-colored-pieces-if-both-neighbors-are-the-same-color.go
func winnerOfGame(colors string) bool {
    alice := 0
    bob := 0
    for i := 1; i < len(colors) - 1; i++ {        
        if colors[i] == 'A' && colors[i-1] == 'A' && colors[i+1] == 'A' {
            alice++
        } else if colors[i] == 'B' && colors[i-1] == 'B' && colors[i+1] == 'B' {
            bob++
        }
    }
    return alice > bob
}

File: 2038-remove-colored-pieces-if-both-neighbors-are-the-same-color/2038-remove-colored-pieces-if-both-neighbors-are-the-same-color.py
class Solution:
    def winnerOfGame(self, colors: str) -> bool:
        a = 0
        b = 0
        for i in range(1, len(colors) - 1):
            if colors[i] == 'A' and colors[i-1] == 'A' and colors[i+1] == 'A':
                a += 1
            elif colors[i] == 'B' and colors[i-1] == 'B' and colors[i+1] == 'B':
                b += 1
        return a > b